faucet/valve.py

```diff
@@ -612,6 +616,15 @@ def _update_stack_link_state(self, ports, now, other_valves):
                     ofmsgs_by_valve[valve].extend(valve.flood_manager.add_vlan(vlan))
                 for port in valve.dp.stack_ports:
                     ofmsgs_by_valve[valve].extend(valve.host_manager.del_port(port))
+
+                if valve.flood_manager.graph and not graph_updated:
```

Comment:
Why is this here -- can this code be placed outside of the for loop?   Are there flood_manages somehow that don't have a graph attached? Would be a lot cleaner conceptually...

Reply:
A cleaner way wit hverification might be to have a separate loop, where we iterate until we find a flood manager with the graph and print it and break. Would that make sense?